velox/docs/functions/presto/datetime.rst

```diff
@@ -188,6 +188,19 @@ Date and Time Functions
 
         SELECT current_timezone;   -- Asia/Kolkata
 
+.. function:: current_timestamp() -> timestamp with time zone
```

Comment:
@anandamideShakyan : We generally order by function names in rst file but I don't see this convention being followed in this file. 

Maybe do a follow up to reorganize these documentation in sorted order of function names.